tests/conftest.py

```diff
@@ -2436,6 +2436,9 @@ def factory(replicas=3, run_in_bg=False, skip_analyze=False):
         couchbase.run_workload(replicas=replicas, run_in_bg=run_in_bg)
         # Run sanity check on data logs
         couchbase.analyze_run(skip_analyze=skip_analyze)
+        # Export data to the google sheet
+        couchbase.export_pfoutput_to_googlesheet(sheet_name='E2E Workloads', sheet_index=2)
```

Comment:
Can we exclude importing the data to gsheet from this fixture?
This fixture can be called for any test(including failure tests) and IMO, there is no need to capture this data for failure tests.

Reply:
@PrasadDesala what index sheet is that?

Reply:
Ignore I got it, If we can also post, pass/fail probably it would be easier to delete those failed runs.

Reply:
if the test is failed, I don't think so code will reach till export to the sheet,
a teardown would happen immediately

Reply:
@vasukulkarni the failure tests I am referring to here are tier-4 tests.